src/utils/bridge/evm/classes/bridge.ts

```diff
@@ -0,0 +1,11 @@
+import type { EvmHistory } from '@sora-substrate/util/build/evm/types';
```

Comment:
what do `classes` mean? I don't see any class here if it's about structures

Reply:
Yep, i forgot to transfer this types to index.ts file, where bridge instance is created